src/views/Home.jsx

```diff
@@ -1,7 +1,23 @@
 import './Home.css';
-import { Link } from 'react-router-dom';
+import { Link, Navigate } from 'react-router-dom';
+import { useState } from 'react';
+import { checkCount } from '../api/firebase';
+
+export function Home({ setListToken, handleNewToken }) {
+	const [inputToken, setInputToken] = useState('');
+	const [message, setMessage] = useState('');
```

Comment:
It might help the reader's understanding if this were a bit more descriptive, such as `notFoundMessage`.